ui/components/NodeInputEmail.tsx

```diff
@@ -13,7 +13,7 @@ export const NodeInputEmail: FC<NodeInputProps> = ({
 }) => {
   const [hasLocalValidation, setLocalValidation] = React.useState(false);
 
-  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  const emailRegex = /^[^\s@]+@[^\s@]+$/;
```

Comment:
See the [discussion in the ticket](https://github.com/canonical/identity-platform-login-ui/issues/385#issuecomment-2545857792).

Reply:
Sorry, hadn't looked at the issue comments. If I understand correctly, this is to cater to obsolete email formats (?)

Reply:
Not sure if they are obsolete, I think they are still valid. But basically: yes :)

Reply:
The reason they are still valid is because of https://datatracker.ietf.org/doc/html/rfc2822#section-3.4.1:
> domain          =       dot-atom / domain-literal / obs-domain

`obs-domain` is defined in https://datatracker.ietf.org/doc/html/rfc2822#section-4.4, no?